glusterd2/commands/volumes/volume-start.go

```diff
@@ -104,22 +107,28 @@ func volumeStartHandler(w http.ResponseWriter, r *http.Request) {
 			UndoFunc: "vol-start.StartBricksUndo",
 			Nodes:    volinfo.Nodes(),
 		},
+		{
+			DoFunc:   "vol-start.UpdateVolinfo",
+			UndoFunc: "vol-start.UpdateVolinfo.Undo",
+			Nodes:    []uuid.UUID{gdctx.MyUUID},
+		},
 	}
 
-	if err := txn.Ctx.Set("volinfo", volinfo); err != nil {
+	if err := txn.Ctx.Set("oldvolinfo", volinfo); err != nil {
 		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err)
```

Comment:
May be we can add logging here.

Reply:
It's already logged internally at `Error` log level.

Reply:
See https://github.com/gluster/glusterd2/blob/v4.1.0/glusterd2/transaction/context.go#L69